tests/qunit/tests/index.js

```diff
@@ -1,15 +1,15 @@
 export const tests = () => {
-	require( './core/common/assets/js/api/modules/command-base.spec.js' );
-	require( './core/common/assets/js/api/modules/command-data.spec.js' );
+	require( './modules/web-cli/assets/js/api/modules/command-base.spec.js' );
+	require( './modules/web-cli/assets/js/api/modules/command-data.spec.js' );
 
-	require( './core/common/assets/js/api/core/commands.spec.js' );
-	require( './core/common/assets/js/api/core/components.spec.js' );
-	require( './core/common/assets/js/api/core/data.spec.js' );
-	require( './core/common/assets/js/api/core/routes.spec.js' );
+	require( './modules/web-cli/assets/js/api/core/commands.spec.js' );
+	require( './modules/web-cli/assets/js/api/core/components.spec.js' );
+	require( './modules/web-cli/assets/js/api/core/data.spec.js' );
+	require( './modules/web-cli/assets/js/api/core/routes.spec.js' );
 
-	require( './core/common/assets/js/api/core/hooks/base.spec.js' );
+	require( './modules/web-cli/assets/js/api/core/hooks/base.spec.js' );
 
-	require( './core/common/assets/js/api/extras/hash-commands.spec' );
+	require( './modules/web-cli/assets/js/api/extras/hash-commands.spec' );
```

Comment:
Maybe create index file for all those tests, that holds only require(s).
`tests.spec.js`

Reply:
Done